### DISCUSSION TITLE
Multi-Workspace-Functionality?

### DISCUSSION THREAD

**Discussion by Der-Feind** [twentyhq/twenty#5685] [Ideas]
Hey Guys,
I've read about a Multi-Workspace-Functionality, but I can't create another one. I'm a Solopreneur and I need 3 Workspaces, as I have 3 different branches... Is this possible?

**Comment by sysaeon** [comment]
If you login/signup with different email accounts, it creates a separate workspace for each account.

It looks like that creates a new schema in the database for each workspace:

```
default=# \dn
                     List of schemas
                Name                 |       Owner       
-------------------------------------+-------------------
 core                                | twenty
 graphql                             | twenty
 metadata                            | twenty
 public                              | pg_database_owner
 workspace_4xnr6z9y1zxsq9ls7uidushoa | twenty
 workspace_5usueorqdy0ul0wk0ioeucln6 | twenty
 workspace_f26m9ur0je2uxwmazuj9z2pg9 | twenty
(7 rows)
```

**Comment by leejuanvn** [comment]
When I create a new workspace, the system reports 'New Workspace is disabled.' Where do I need to enable it?

**Comment by BOHEUS** [comment]
@leejuanvn please create an issue with video and steps to reproduce the problem as well as information such as version of Twenty, cloud or self-hosted to help solve the problem

**Comment by BOHEUS** [comment]
To create new workspace, you need to add `IS_MULTIWORKSPACE_ENABLED=true` in .env, then this option should be available 

<img width="1680" height="1014" alt="image" src="https://github.com/user-attachments/assets/032b67b3-b5bd-42ca-9df5-ae7508dde789" />
